includes/uninstall.php

```diff
@@ -154,6 +154,7 @@ function delete_posts(): void {
 		[
 			'fields'           => 'ids',
 			'suppress_filters' => false,
+			'post_status' => 'any',
```

Comment:
Needs proper arrow alignment.

Tip: run `npx husky install` to set up git pre-commit hooks that catch this kinda stuff before pushing.

Reply:
I seem to run into this depending on which PHP version I'm running doesn't catch on PHP 8.x .... fine when using 7.x

Reply:
I see. PHPCS PHP 8.x compatbility is not fully released yet unfortunately